idn/idna_convert.class.php

```diff
@@ -1,969 +0,0 @@
-<?php
-// {{{ license
```

Comment:
Removing this file might be a breaking change. What do you think about keeping this file and simply triggering a deprecation error?

```php
<?php

declare(strict_types=1);

trigger_error('Using class `idna_convert` from `idn/idna_convert.class.php` is deprecated since SimplePie 1.9.0, use native `idn_to_ascii()` function instead.', \E_USER_DEPRECATED);
```

Reply:
I would argue that this is not a part of API – it is not part of SimplePie, it is not listed in the API docs nor was it documented anywhere else.

I have listed it in the Removed section of changelog so that if someone uses it for themselves, they can just copy it from older version.